### DISCUSSION TITLE
Download Development Build

### DISCUSSION THREAD

**Discussion by Cheerfulbull** [ihhub/fheroes2#7523] [Q&A]
This is probably one of these silly annoying questions, but is there any way for one to get a dev. build? I want to try out the editor, but it doesn't even work on linux. When I try to build, make doesn't compile the editor, and CMake tells me that the cmake file has a syntax error. Any hints?

**Comment by ihhub** [comment]
Hi @Cheerfulbull , the Editor is not ready at all and it does nothing at the moment. Therefore it is solely for the developers. There is no such a thing as compiling the Editor. It is a part of the main application.

Also be specific and provide the exact instructions you followed and commands you used in order to attempt to compiling the Editor.

**Comment by Mr-Bajs** [comment]
@Cheerfulbull The latest dev builds are available here, snapshot if I understand correctly.
https://ihhub.github.io/fheroes2/INSTALL.html

**Reply by zenseii** [reply]
Hi, @Mr-Bajs. Those are actually snapshot builds, which are not the same as dev builds. To get the dev build you have to build it yourself. We have instructions on how to do that.

**Reply by oleg-derevenetz** [reply]
@Mr-Bajs more specifically, the dev build has additional debugging/monitoring facilities turned on - e.g. you are able to see the entire map, get additional info about tiles/objects in the debug console, transfer the player control from a human to the AI and regain it back, etc, etc. Snapshot builds have all this turned off, it's a regular builds, just for the latest commit.
